packages/base/src/commands.ts

```diff
@@ -1140,6 +1144,36 @@ export function addCommands(
     }
   });
 
+  commands.addCommand(CommandIDs.getGeolocation, {
```

Comment:
Can you think of a scenario where we'd want to disable this button? In those cases we'd want to handle those in `isEnabled`

Reply:
Yes. In a version where the geolocation would be materialized by a marker on the map, we can imagine that we the command is enabled only where there is no marker on the map yet. Or we could imagine a version with a toogle button or a toggable custom layer. In any case, this PR addresses a first simple geolocation feature that will be improved in a future PR.